controllers/front/listener.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+class ps_qualityassurancelistenerModuleFrontController extends ModuleFrontController
```

Comment:
```suggestion
class Ps_QualityAssuranceListenerModuleFrontController extends ModuleFrontController
```

Reply:
I followed some documentation to build a front controller and it was lowercase only 🤔  . But I agree this is better.

Reply:
Fixed in https://github.com/PrestaShop/ps_qualityassurance/pull/12/commits/9e71f01f2a08b7fc527126e0a83d1a7ea458eb0c